### PR TITLE
[teamd]: Add teammgrd in docker-teamd

### PR DESCRIPTION
**Pull request by stcheng**
Remove the teamd.j2 templates used for starting the teamd. Add
teammgrd instead to manage all port channel related configuration
changes. Remove front panel port related configurations in
interfaces.j2 templates as well.

Update the sonic-swss submodule.

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

**Comment by lguohan** [comment]
there conflict.

File: dockers/docker-teamd/supervisord.conf [hunk 1]
```diff
@@ -3,24 +3,24 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
-[program:start.sh]
```
Comment (jleveque) [inline dockers/docker-teamd/supervisord.conf]: Why remove start.sh here? It still seems necessary.

File: dockers/docker-teamd/Dockerfile.j2 [hunk 2]
```diff
@@ -25,7 +25,7 @@ RUN dpkg -i \
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
-COPY ["start.sh", "teamd.sh", "/usr/bin/"]
+COPY ["teamd.sh", "/usr/bin/"]
```
Comment (jleveque) [inline dockers/docker-teamd/Dockerfile.j2]: Why remove start.sh here? It still seems necessary. And why add teamd.sh, as it appears you deleted the file in this PR?

Comment (stcheng) [reply dockers/docker-teamd/Dockerfile.j2]: thanks was to remove the file teamd.sh

**Comment by stcheng** [comment]
updated the sonic-swss submodule as well.

**Pull request merged**